app/controllers/nonsipp/memberdetails/upload/FileUploadErrorSummaryController.scala

```diff
@@ -78,7 +78,7 @@ object FileUploadErrorSummaryController {
       heading = "fileUploadErrorSummary.heading",
       description = Some(
         ParagraphMessage("fileUploadErrorSummary.paragraph") ++
-          Heading2("fileUploadErrorSummary.heading2", LabelSize.Medium) ++
+          Heading2("fileUploadErrorSummary.heading2", HeadingSize.Medium) ++
```

Comment:
These heading size related changes don't seem related to the tasklist, is this fixing some existing issue? Or just making the code more flexible?

Reply:
This was some of the custom CSS that broke after the library update, so I fixed it as part of that same commit.